### DISCUSSION TITLE
关于SSR Plus+使用过程中的Unable to launch the requested CGI program:   /www/cgi-bin/luci: No such file or directory报错, 有无解决方法

### DISCUSSION THREAD

**Discussion by umlka** [coolsnowwolf/lede#11014] [Bugs / 问题反馈]
在/cgi-bin/luci/admin/services/shadowsocksr/servers界面节点连接测试过程中手动切换到/cgi-bin/luci/admin/services/shadowsocksr/client界面时, luci直接boom掉, 可以复现
固件为github action最新版本, x86 efi固件

**Comment by ramondelee** [comment]
如何解决的，我也遇见了
